clean_project/search.py

```diff
@@ -0,0 +1,60 @@
+"""
+Search a word in a file and return all the occurrence specified the line numbers
+"""
+import argparse
+import psycopg2
+
+BOLD = '\033[1m'
+END = '\033[0m'
+
+
+def search(search_word):
+    """
+    Search if a given word is in bucket of files
+    :param search_word: Word given to search
+    :return: List of files and numbers of file when search_word occurrence
+    """
+    connection = None
+    result_search = []
+    try:
+        connection = psycopg2.connect(database='index_db', user='gianluca', password='password')
```

Comment:
not sure why here you "regressed" your solution by removing the object instantiation. this way you need to open a connection to the database for each request. isn't that similar to the inefficiency you had previously with the loading of the json index for each query?